extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -30,15 +31,19 @@
     public Optional<String> applicationServer;
 
     /**
-     * A comma-separated list of topic names processed by this stream processing application.
+     * A comma-separated list of topic names.
      * The pipeline will only be started once all these topics are present in the Kafka cluster.
      */
     @ConfigItem
-    public Optional<String> topics;
+    public List<String> topics;
 
     @Override
     public String toString() {
         return "KafkaStreamsRuntimeConfig [applicationId=" + applicationId + ", bootstrapServers=" + bootstrapServers
                 + ", applicationServer=" + applicationServer + ", topics=" + topics + "]";
     }
+
+    public List<String> getTrimedTopics() {
```

Comment:
Sorry to be picky but I think it should be `getTrimmedTopics()` with 2 m.

Could you fix that and amend your commit? Then I think we are good to go!

Reply:
Damn, you're right.

It's fixed now.
Thanks for review.